### ISSUE TITLE
Support for graph settings

### ISSUE THREAD

**Issue by wchrisjohnson** [honeycombio/terraform-provider-honeycombio#103]
We would like to see full support for graph and board settings in the provider.

Query related items:
- useStackedGraphs
- omitMissingValues

Board related items:
- one column vs multi-column

**Is your feature request related to a problem? Please describe.**
Most/all of our boards are created via Terraform, and these are blockers for us atm.

**Describe the solution you'd like**
Support for the above.

**Describe alternatives you've considered**
We are currently creating boards by hand and planning to go back and migrate to terraform once these features (and others) land.

**Additional context**
None.

**Comment by jharley** [comment]
Thanks for opening this, @wchrisjohnson! This is actually a duplicate of #37 so I'm going to close it in favour of the original issue and ask that you follow along with that one.

**Comment by wchrisjohnson** [comment]
@jharley That's fine by me as long as we don't lose the additional requested item (not graph related) board setting "one column vs multiple cols". Let me know if you want a new request, or reopen this one just for that.

**Comment by jharley** [comment]
@wchrisjohnson opened #109 to track board one column vs multiple column display
